src/test/java/org/cactoos/http/HtHeadTest.java

```diff
@@ -143,18 +144,20 @@ public void edgeOfTheBlockTearing() throws IOException {
                 new HtHead(
                     new InputOf(
                         new JoinedText(
-                            "\r\n",
+                            new TextOf("\r\n"),
                             block,
-                            "",
-                            "text here"
+                            new TextOf(""),
+                            new TextOf("body here")
                         )
                     )
                 )
-            ).asString(),
-            Matchers.allOf(
-                new StringStartsWith("HTTP"),
-                new StringContains("OK\r\nReferer"),
-                new StringEndsWith("text/plain")
+            ),
+            new TextIs(
+                Matchers.allOf(
+                    new StringStartsWith("HTTP"),
+                    new StringContains("OK\r\nReferer"),
+                    new StringEndsWith("text/plain")
+                )
```

Comment:
@ilyakharlamov when cactoos-matches is released, please use directly `allOf` with Text based matchers (instead of using `TextIs` which makes all of this akward :)